physionet-django/project/migrations/0062_auto_20220221_0347.py

```diff
@@ -0,0 +1,162 @@
+        ),
+        migrations.AlterModelOptions(
+            name='author',
+            options={'default_permissions': ()},
+        ),
+        migrations.AlterModelOptions(
+            name='authorinvitation',
+            options={'default_permissions': ()},
+        ),
+        migrations.AlterModelOptions(
+            name='contact',
+            options={'default_permissions': ()},
+        ),
+        migrations.AlterModelOptions(
+            name='copyeditlog',
+            options={'default_permissions': ()},
+        ),
+        migrations.AlterModelOptions(
+            name='coreproject',
+            options={'default_permissions': ()},
+        ),
+        migrations.AlterModelOptions(
+            name='dataaccess',
+            options={'default_permissions': ()},
+        ),
+        migrations.AlterModelOptions(
+            name='dataaccessrequest',
+            options={'default_permissions': ()},
+        ),
+        migrations.AlterModelOptions(
+            name='dataaccessrequestreviewer',
+            options={'default_permissions': ()},
+        ),
+        migrations.AlterModelOptions(
+            name='documenttype',
+            options={'default_permissions': ()},
+        ),
+        migrations.AlterModelOptions(
+            name='duasignature',
+            options={'default_permissions': ()},
+        ),
+        migrations.AlterModelOptions(
+            name='editlog',
+            options={'default_permissions': ()},
+        ),
+        migrations.AlterModelOptions(
+            name='gcp',
+            options={'default_permissions': ()},
+        ),
+        migrations.AlterModelOptions(
+            name='legacyproject',
+            options={'default_permissions': ()},
+        ),
+        migrations.AlterModelOptions(
+            name='programminglanguage',
+            options={'default_permissions': ()},
+        ),
+        migrations.AlterModelOptions(
+            name='projecttype',
+            options={'default_permissions': ()},
+        ),
+        migrations.AlterModelOptions(
+            name='publication',
+            options={'default_permissions': ()},
+        ),
+        migrations.AlterModelOptions(
+            name='publishedaffiliation',
+            options={'default_permissions': ()},
+        ),
+        migrations.AlterModelOptions(
+            name='publishedauthor',
+            options={'default_permissions': ()},
+        ),
+        migrations.AlterModelOptions(
+            name='publishedproject',
+            options={
+                'default_permissions': ('change',),
+                'permissions': [
+                    ('can_edit_featured_content', 'Can edit featured content'),
+                    ('can_view_access_logs', 'Can view access logs'),
+                    ('can_view_project_guidelines', 'Can view project guidelines'),
+                    ('can_view_stats', 'Can view stats'),
+                    ('can_assign_editor', 'Can assign editor')
```

Comment:
Typically editors are assigned to active projects (i.e. pre-publication projects) so that the editor can review the content prior to publication. Should `can_assign_editor` be assigned to ActiveProject instead?

Reply:
[21b0929](https://github.com/MIT-LCP/physionet-build/pull/1487/commits/21b0929c3feab795173a69ac1640e5ecb9ddae98)

[3674e0b](https://github.com/MIT-LCP/physionet-build/pull/1487/commits/3674e0b706d1fcc7fa0d7980b44eb228e9ede865)